### PR TITLE
Branch populate exercises

### PR DESCRIPTION
**Pull request by a1021492980**
Add ExerciseList class to access and populate exercise from local file and created folder to store exercises locally.

### CODE REVIEW

File: src/main/java/seedu/duke/ExerciseList.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    private static final String EXERCISE_FILE_PATH = "exercises/Exercises.txt";
+    private static final File exerciseFile = new File(EXERCISE_FILE_PATH);
+    private final ArrayList<String> exercises = new ArrayList<>();
+    private int numberOfExercises = 0;
+
+    /**
+     * Reads the exercises from local file and store them into an ArrayList named exercises.
+     * @throws FileNotFoundException when the local file is missing.
+     */
+    protected void loadExercises() throws FileNotFoundException {
```
Comment (Musfirahe0556596) [inline src/main/java/seedu/duke/ExerciseList.java:18]: Perhaps can add throw IOException?

File: src/main/java/seedu/duke/ExerciseList.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            var exercise = scanner.nextLine();
+            exercises.add(exercise);
+            numberOfExercises += 1;
+        }
+    }
+
+    /**
+     * Prints the list of exercises starting with their index.
+     */
+    protected void exercisesList() {
```
Comment (Musfirahe0556596) [inline src/main/java/seedu/duke/ExerciseList.java:30]: Perhaps a suggested method name could be printExerciseList()

File: src/main/java/seedu/duke/ExerciseList.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        }
+    }
+
+    /**
+     * Returns the exercise at given index.
+     * @param index index of exercise.
+     * @return the exercise at given index.
+     * @throws IndexOutOfBoundsException when the index provided is less than 0 or greater than the numberOfExercises.
+     */
+    protected String getExerciseAt(int index) throws IndexOutOfBoundsException{
```
Comment (Musfirahe0556596) [inline src/main/java/seedu/duke/ExerciseList.java:42]: Just a minor formatting issue - add a white space after IndexOutOfBoundsException and the curly bracket { !

**Review by Musfirahe0556596 — COMMENTED** [review]
Looks good! Just minor code quality issues!

**Comment by alanlowzies** [comment]
Other than what @Musfirahe0556596 has mentioned, I think the rest is fine for now. We can iron out the specifics down the road.

**Comment by alanlowzies** [comment]
Java CI build failure issue raised in CS2113T forum: https://github.com/nus-cs2113-AY2122S2/forum/issues/44

**Comment by alanlowzies** [comment]
Fixes #7

**Comment by Musfirahe0556596** [comment]
Fixes #7

**Pull request merged**